### PR TITLE
Add rollout#multiget

### PR DESCRIPTION
**Pull request by akhiln**
Add a method to fetch multiple features at once from Redis. This will help when trying to determine what flags are active for a specific user.

### CODE REVIEW

File: spec/rollout_spec.rb [hunk 1]
```diff
@@ -306,6 +306,29 @@
...
+    it "returns an array of features" do
+      features = @rollout.multi_get(:chat, :videos, :signup)
+      features[0].name.should == :chat
+      features[0].groups.should == [:caretakers]
+      features[0].percentage.should == 10
+      features[1].name.should == :videos
+      features[1].groups.should == [:greeters]
+      features[2].name.should == :signup
+      features[2].percentage.should == 100
+      features[0].name.should == :chat
```
Comment (TrangPham) [inline spec/rollout_spec.rb]: you're checking `features[0].name` twice. I think you meant to check `features[2].groups` here?

Comment (akhiln) [inline spec/rollout_spec.rb]: You're right, its an extra line. Removed.

File: lib/rollout.rb [hunk 2]
```diff
@@ -180,21 +180,21 @@ def active_in_group?(group, user)
...
+  
+  def multi_get(*features)
+    feature_keys = features.map{|feature| key(feature) }
+    feature_strings = @storage.mget(*feature_keys)
+    collection = []
+    feature_strings.each_with_index do |string, index|
+      feature = features[index]
+      collection << load_feature(feature, string)
+    end
+    collection
```
Comment (snags88) [inline lib/rollout.rb]: this might be cleaner if you use `feature_strings.map.with_index`. Then you can get rid of the collection variable

**Comment by reneklacan** [comment]
Added in https://github.com/fetlife/rollout/pull/105

**Pull request closed without merging**